CHANGELOG.md

```diff
@@ -4,12 +4,47 @@
 
 ## [Unreleased] - ReleaseDate
 
-- Significantly optimize cache updates (first update will still be a full update)
-- Replace escape menu modding tab with mod integration interface
+### General
+
+- Fix unintentionally linking to libssl on Linux. This used to prevent some users on various Linux
+  distros from being able to launch mint at all.
+
+### User Interface
+
 - Add light/dark mode toggle to settings menu
-- More mod save file fixes for Windows store version
+- Replace escape menu modding tab with new modding menu
+- Show mint mods in public server list
+- Show time since last action
+- Make mod URL searchable for mods without cache data
+- Implement load priority to no longer rely on implicit ordering
+- Add mod list sorting
+- Slightly improved error reporting; mint should now indicate the mod that caused a failure
+- Various GUI improvements
+
+### Core Functionality
+
+- Implement self-update
+- Fix mod url resolution
+- Fix mods sometimes integrating in incorrect order
+- Add patch to fix gas clouds not exploding sometimes
+- Some mod save file fixes for Windows store version
+
+### Internal Changes
+
+- Significantly optimize cache updates (first update will still be a full update)
+- Allow overriding appdata dir via CLI flag
+- Rename cache and config directories from `drg-mod-integration` to `mint` and default to legacy
+  if they exist
 - Fix Windows console being full of garbage characters
-- Fix unintentionally linking to libssl on Linux
+- Disable LTO for dev builds
+- Add nix flake
+- Improved testing
+- Various CI improvements
+- Try to enable ANSI code support on Windows 10
+- Reduce release size by removing debug symbols
+- Package Linux target with zip instead of tar
+- Various Asset Registry related fixes
```

Comment:
Since asset registry was not present at all in last release I would consider asset registry population a new feature rather than just a fix

Reply:
Moved this to core functionality.